packages/notification-services-controller/package.json

```diff
@@ -67,8 +66,7 @@
     "typescript": "~4.9.5"
   },
   "peerDependencies": {
-    "@metamask/keyring-controller": "^17.0.0",
-    "@metamask/profile-sync-controller": "^0.0.0"
```

Comment:
This is temporary so linting passes.

We will add this back in once we release the notification-services-controller